evap/results/exporters.py

```diff
@@ -160,18 +160,22 @@ def filter_evaluations(semesters, evaluation_states, degrees, course_types, cont
         return evaluations_with_results, used_questionnaires, course_results_exist
 
     def write_headings_and_evaluation_info(
-        self, evaluations_with_results, semesters, contributor, degrees, course_types
+        self, evaluations_with_results, semesters, contributor, degrees, course_types, verbose_heading
     ):
         export_name = "Evaluation"
         if contributor:
             export_name += f"\n{contributor.full_name}"
         elif len(semesters) == 1:
             export_name += f"\n{semesters[0].name}"
-        degree_names = [degree.name for degree in Degree.objects.filter(pk__in=degrees)]
-        course_type_names = [course_type.name for course_type in CourseType.objects.filter(pk__in=course_types)]
-        self.write_cell(
-            _("{}\n\n{}\n\n{}").format(export_name, ", ".join(degree_names), ", ".join(course_type_names)), "headline"
-        )
+        if verbose_heading:
+            degree_names = [degree.name for degree in Degree.objects.filter(pk__in=degrees)]
+            course_type_names = [course_type.name for course_type in CourseType.objects.filter(pk__in=course_types)]
+            self.write_cell(
+                _("{}\n\n{}\n\n{}").format(export_name, ", ".join(degree_names), ", ".join(course_type_names)),
+                "headline",
+            )
+        else:
+            self.write_cell(_("{}").format(export_name))
```

Comment:
```suggestion
            self.write_cell(("{}").format(export_name), "headline")
```

You removed the format description, so the styling was wrong

Reply:
(now an updated version, incorporating the other changes, would be
```python3
self.write_cell(export_name, "headline")
```
)